distribution/docs/src/main/resources/content/_running/application-commands.adoc

```diff
@@ -0,0 +1,178 @@
+:title: Application Commands
+:type: subMaintaining
+:status: published
+:parent: Available System Console Commands
+:summary: Application commands available.
+:order: 01
+
+== {title}
+
+Application commands are used from the ${ddf-admin} application to manage applications in the ${branding}.
+
+[NOTE]
+====
+The Application Commands are installed automatically with the ${ddf-admin} Application.
+====
+
+.[[app_command_descriptions]]App Command Descriptions
+[cols="1m,1m,5" options="header"]
+|===
+
+|Command
+|Syntax
+|Description
+
+|add
+|app:add appUri
+|Adds an application with the given uri.
+
+|remove
+|app:remove appName
+|Removes an application with the given name.
+
+|start
+|app:start appName
+|Starts an application with the given name.
+
+|stop
+|app:stop appName
+|Stops an application with the given name.
+
+|list
+|app:list
+|Lists the applications that are in the system and gives their current state.
+
+|status
+|app:status appName
+|Shows status of an application.
+Gives information on the current state, features within the application, what required features are not started and what required bundles are not started.
```

Comment:
As a nit, the last part of this sentence seems clunky.  `Gives information on the current state, features within the application, and what required features and bundles are not started.`

Reply:
agreed. Fixed.